packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

```diff
@@ -113,63 +126,77 @@ To complete the one-time verification of the Event Hook:
 
-    > **Note:** If you are using the preview sample data, you must edit the preview to add a `target` value, including an email using the `alternateId` property. For example, `"target": [{ "alternateId": "john.doe@example.com"}]`. Otherwise, the sample application code throws an error if the value in `target` property is `null`.
+    > **Note:**
+    >
+    > * The preview Event Hook JSON body can be modified for testing or development purposes. Click **Edit** to update the `"target"` field, for example.
+    > * If you are using the preview sample data, you must edit the preview to add a `target` value, including an email using the `alternateId` property. For example, `"target": [{ "alternateId": "john.doe@example.com"}]`. Otherwise, the sample application code throws an error if the value in `target` property is `null`.
 
 1. Ensure that both your ngrok session and local sample application are running.
 
-    >**Note:** If you start a new ngrok session at any time, make sure to update the Event Hook URL.
+    > **Note:** If you start a new ngrok session at any time, make sure to update the Event Hook URL.
 
 1. Click **Deliver Request**. The Event Hook Preview displays the status request as either successful or a failure. Check your local application console. The following message appears if successful:
 
      `The user john.doe@example.com has been added to the Okta org!`
 
-1. Check your ngrok inspection interface (`http://localhost:4040`). Each call recorded by ngrok appears in the interface from which you can review the complete call response body, header, and other details.
+1. Check your ngrok inspection interface (`http://localhost:4040`). Each call recorded by ngrok appears in the interface from which you can review the response body, header, and other details.
 
     ![A screen shot of the ngrok inspection interface that includes details of a call.](/img/ngrok-and-event-hooks-web-interface-small.png)
 
-### Test
+### Replay the Event Hook
 
-To run a test of your Event Hook:
+The ngrok inspection interface provides a replay functionality that you can use to test your code without reproducing test conditions in Okta. To use the replay:
 
-1. Ensure that both your ngrok session and local sample application are running.
-1. In your Okta org, sign in as an administrator and create a test user in the Admin Console.
-    * Go to **Directory** > **People**, and click **Add Person**. As an example, add the user Jane Doe with the following fields:
-        * **First Name:** Jane
-        * **Last Name:** Doe
-        * **User Name:** jane.doe@example.com
+1. In ngrok’s inspection interface (http://localhost:4040), select an event hook from Okta.
```

Comment:
Not sure if "event hook" should be capitalize here?

Reply:
Yes, thanks.